### DISCUSSION TITLE
[HELP] Plotting at 100% but can'tcopy to my Final directory (8tb free space, exFAT)

### DISCUSSION THREAD

**Discussion by r1235613** [Chia-Network/chia-blockchain#3181] [General]
**Describe the bug**
`
Could not copy "/chia_ssd/plot-k32-2021-04-30-01-49-[xxxx].plot.2.tmp" to "/wdbook/chia_plots/plot-k32-2021-04-30-01-49-[xxxx].plot.2.tmp". Error Operation not permitted. Retrying in five minutes.
`

**Additional context**
I can use touch/rm command to create/delete file at final directory.
I use non-root account to plotting.
My final directory is 12 TB USB external HDD, format by exFAT.
I mount HDD by option 'umask=000,dmask=000'.
`drwxrwxrwx  2 root root 1048576 Apr 30 13:00  chia_plots`

How can I find detail log to resolve this problem?

**Comment by jsanchezcorral** [comment]
I have the same error, please help

**Comment by moshenskyDV** [comment]
I've tried to do plotting on linux. Half of disks I formatted to NTFS and second half to exFAT. NTFS plotted normally, but exFAT stuck with the same permission error as yours. 
So I formatted everything to NTFS, it doesn't require root access. And with NTFS  everything works fine now. Also I can use this disks in other windows machine.
